### PR TITLE
Part 10: scope cross_logical_cluster tests

### PR DESCRIPTION
**Pull request by varshaprasad96**
To verify if e2e tests pass in CI

## Related issue(s)

Fixes #

### CODE REVIEW

File: test/e2e/conformance/cross_logical_cluster_list_test.go [hunk 1]
```diff
@@ -114,7 +113,7 @@ func bootstrapCRD(
 	client apiextensionsv1client.CustomResourceDefinitionInterface,
```
Comment (sttts) [inline test/e2e/conformance/cross_logical_cluster_list_test.go:113]: nit: technically clusterClient

File: test/e2e/watchcache/watchcache_enabled_test.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ func TestWatchCacheEnabledForCRD(t *testing.T) {
 	crdClient := apiExtensionsClients.Cluster(cluster).ApiextensionsV1().CustomResourceDefinitions()
 
 	t.Log("Creating wildwest.dev.cowboys CR")
-	wildwest.Create(t, crdClient, cowBoysGR)
+	wildwest.Create(t, logicalcluster.Name{}, crdClient, cowBoysGR)
```
Comment (sttts) [inline test/e2e/watchcache/watchcache_enabled_test.go:71]: cluster, not empty.

Comment (varshaprasad96) [reply test/e2e/watchcache/watchcache_enabled_test.go:71]: This is temporary. I'll create a follow up PR to modify clients in `watchcache/` and that will make sure to pass the right cluster name while creating the object.

Comment (sttts) [reply test/e2e/watchcache/watchcache_enabled_test.go:71]: ack

**Pull request merged**